### PR TITLE
cheatsheet_plotly

### PR DESCRIPTION
**Pull request by siqihe41099**
cheatsheet_plotly

### CODE REVIEW

File: plotly_cheatsheet.Rmd [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+
+
+
+
+1. Histogram
```
Comment (Francis-ZY-Yuan) [inline plotly_cheatsheet.Rmd:26]: Use `##` for subheadings, instead of numbers.

File: plotly_cheatsheet.Rmd [hunk 2]
```diff
@@ -0,0 +1,145 @@
+---
```
Comment (Francis-ZY-Yuan) [inline plotly_cheatsheet.Rmd:1]: Rmd should not contain `---` lines, and should start with a `#` followed by a space and your chapter name.

**Review by Francis-ZY-Yuan — CHANGES_REQUESTED** [review]

File: plotly_cheatsheet.Rmd (hunk 2 above)
Comment (siqihe41099) [reply plotly_cheatsheet.Rmd]: Hello, I uploaded the latest version. Do you have time to take a look?

**Comment by Francis-ZY-Yuan** [comment]
Merged into main. Thanks!

**Pull request merged**